new-pipeline/dist.html

```diff
@@ -14,19 +14,18 @@
     <link rel="stylesheet" type="text/css" href="lib/daterangepicker-bs3.css" />
     <link rel="stylesheet" type="text/css" href="style/elessar.css" />
     <link rel="stylesheet" type="text/css" href="style/metricsgraphics.css" />
-
     <link rel="stylesheet" type="text/css" href="style/dashboards.css" />
     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-49796218-2"></script>
 </head>
 <body>
     <div class="container-fluid">
         <header>
             <form class="navbar-form inline pull-right">
+                <span class="permalink-control"></span>
```

Comment:
Great, that already makes it better!
Is it easy to add a title to it too in this PR?
It looks like this will require diving into JS in dashboards.js.